xml/System.IO.Compression/BrotliStream.xml

```diff
@@ -367,9 +367,22 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Asynchronously releases the unmanaged resources used by the <see cref="T:System.IO.Compression.BrotliStream" />.</summary>
+        <returns>A task that represents the asynchronous dispose operation.</returns>
+        <remarks>
+          <format type="text/markdown">
+  <![CDATA[
```

Comment:
It might be useful to add brotlistream specific documentation rather than just copying the contents of `stream.DisposeAsync`.

For instance, looking at the impl (https://github.com/dotnet/corefx/blob/306fe9d60cc6db02a69d8ea6b88dde2b98fe8a7c/src/System.IO.Compression.Brotli/src/System/IO/Compression/BrotliStream.cs#L76-L97), its probably useful to mention what happens with `leavOpen` true/false and when the mode is `CompressionMode.Compress`, as remarks.

Reply:
@stephentoub, what do you think?

Reply:
I'd be fine with that.  Note, though, that this same pattern is employed elsewhere, e.g. DeflateStream.  If we're going to add such details here, we should add it in other similar places.